src/main/java/homework10/MyHashMap.java

```diff
@@ -0,0 +1,169 @@
+package homework10;
+
+import java.util.Arrays;
+import java.util.Map;
+
+public class MyHashMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] arrayOfNodes;
+    private int size;
+
+    public V get(K key) {
+        if (getNode(key)== null) {
+            return null;
+        }
+        return getNode(key).getValue();
+    }
+
+    protected V put(K key, V value) {
```

Comment:
this method is difficult to understand. Try to simplify it, for example we can create few small method

Reply:
made small rebuilding. maybe now it is more simple. check it please.